ACloudGuru/serverless-plugin-aws-alerts
# Add omitDefautlDimension param

## What did you implement:

Support a `omitDefaultDimension` parameter at the `alarm` level to avoid the `FunctionName` dimension that's added by default. This will allow the creation of alarms that aren't under the `AWS/Lambda` (API Gateway or SQS for example), which don't require the `FunctionName` dimension. This would address the following issues:
- #104
- #85 
- #19 

## How can we verify it:

Given that this is a relatively trivial change there's not much to verify. However, if you want to try creating an API Gateway alert you can use the following yaml configuration:
```yaml
 - name: function5XX
        description: '5XX response codes'
        namespace: 'AWS/ApiGateway'
        metric: 5XXError
        omitDefaultDimension: true
        dimensions:
          - Name: ApiName
            Value: api
          - Name: Resource
            Value: /api/endpoint
          - Name: Method
            Value: POST
          - Name: Stage
            Value: prod
        threshold: 5
        statistic: Sum
        period: 60
        evaluationPeriods: 1
        datapointsToAlarm: 1
        comparisonOperator: GreaterThanOrEqualToThreshold
```

If you check the alarm in CloudWatch you'll see that it doesn't include the `FunctionName` dimension.

## Todos:
